src/Microsoft.PowerShell.Commands.Utility/commands/utility/UnblockFile.cs

```diff
@@ -121,7 +121,12 @@ protected override void ProcessRecord()
                     }
                     catch (Win32Exception accessException)
                     {
-                        WriteError(new ErrorRecord(accessException, "RemoveItemUnauthorizedAccessError", ErrorCategory.PermissionDenied, path));
+                        // NativeErrorCode=2 - File not found.
+                        // If the block stream not found the 'path' was not blocked and we successfully return.
+                        if (accessException.NativeErrorCode != 2)
```

Comment:
@jpsnover suggests adding a verbose message here if the block stream is not found

Reply:
Done. I tried to avoid adding a resource string

Reply:
I think it would be better to add a resource string saying:

```none
File '{0}' did not contain a 'Zone.Identifier' file stream and was already not blocked.
```

Reply:
I added a resouce string with {0} at last position - we should take into account long paths.